### ISSUE TITLE
JS error when bug assigned to unknown user

### ISSUE THREAD

**Issue by xxyy** [mantisbt-plugins/Taskodrome#28]
Hi,

in Chrome on MantisBT 1.3.0-rc2 with your rc1 branch, I am getting the following JavaScript error in my Chrome Developer Tools console:

`devs_grid.js:253 Uncaught TypeError: Cannot read property 'splice' of undefined`

Because of this, no boards show at all for this project. They are shown for other projects.

These are the relevant lines:

 ````javascript
 for(var i = 0; i != issues_raw.length; ++i) { #i = 21
    var index = idsIndexes[issues_raw[i].handler_id]; //index = undefined, idsIndexes= [0,2:3,10:1,11:2,17:4,28:5]
    issues[index].splice(issues[index].length, 0, issues_raw[i]); // <-- breakpoint here - line 253
  }
````

I have placed a breakpoint on that line and execution went well until the state I added as comments above was reached.

Judging from the part of the code I have read, this is probably related to my installation having some bugs still assigned to somebody who left our team and has been removed from the developers group as a result. Taskodrome still tries to show their bugs in their column, but can't because they don't have a column.

PS: Amazing plugin! I always wanted Scrum boards for Mantis and now I have them! Thanks a lot for this!

Best regards

**Comment by AuthenticEshkinKot** [comment]
@xxyy 
Thanks for detailed description, I'll fix it soon!

**Comment by AuthenticEshkinKot** [comment]
@xxyy 
I've fixed this bug - #34

**Comment by xxyy** [comment]
Confirmed as fixed on my instance. Thanks a lot!
